src/lib.rs

```diff
@@ -209,14 +283,16 @@ pub fn process_root() -> Result<()> {
 // (It probably wouldn't be hard to unify the two functions into a
 //  single method on the `Context`, though.)
 
-fn process_with_stamp(stamp: File) -> Result<()> {
+fn process_with_stamp(stamp: File, emit_rerun_if: bool) -> Result<()> {
+    println!("\n\nemit rerun if: {:?}\n\n", emit_rerun_if);
```

Comment:
can you make this a `debug!` instead? Or just comment it out for now?

Reply:
(same for the other occurrences of `println!` that just look like debug output; I won't bother making a note on each one.)

Reply:
I don't know why I asked for this; all of tango uses println's all over the place!